install-wings.sh

```diff
@@ -535,8 +494,12 @@ function main {
   echo "* This script is not associated with the official Pterodactyl Project."
   echo "*"
   echo "* Running $OS version $OS_VER."
+  echo "* Latest pterodactyl wings version is $WINGS_VERSION"
   print_brake 70
 
+  # checks if the system is compatible with this installation script
+  check_os_comp
+
```

Comment:
Why did you move this?

Reply:
So that it tells what OS it sees and only then stops the script. Then user can see what OS is running. Of course it also show with print_errors. Now I don't know what to say.

Reply:
I guess it's just as good if it's here. I'll leave it.